### ISSUE TITLE
Download All

### ISSUE THREAD

**Issue by johnny-vikrant** [Paari/photos#5]
Would it possible to have a 'download all' photos option.
Thanks.

**Comment by ghost** [comment]
This would require to zip that folder and send back to user browser. 
The same way that Google Drive does. But it also require extra hard disk space to store that zip file.

**Comment by Paari** [comment]
Yeps guys @austinankur @johnny-vikrant the main purpose was to save bandwidth and people only downloading the images they want to download, as the total size of images was over 1.5GB and when we shared last time it was around 5GB.

I am thinking of a feature of selecting particular images and also a select all button, this will download the selected images as a zip file.

**Comment by ghost** [comment]
What about having an option to add selected picture to user's Google Drive.

Google Drive JS API 
https://developers.google.com/drive/v3/web/quickstart/js

**Comment by vikrantnegi** [comment]
Or maybe multi-select photos and then downloading only those would make life easier. .

**Comment by Paari** [comment]
@vikrantnegi yes that is the direction I am thinking of taking, selecting multiple photos and downloading a zip of those with a single click.
